feature-libs/customer-ticketing/components/customer-ticketing-close/customer-ticketing-close.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CustomerTicketingCloseComponent } from './customer-ticketing-close.component';
+
+describe('CustomerTicketingCloseComponent', () => {
+  let component: CustomerTicketingCloseComponent;
+  let fixture: ComponentFixture<CustomerTicketingCloseComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ CustomerTicketingCloseComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CustomerTicketingCloseComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Yes, adding the unit tests.

Reply:
just to understand, you'll add unit test now or later? 

Because I recall the previous PR had no unit tests, but was said it will be done later https://github.com/SAP/spartacus/pull/16005/files#r931180804

Reply:
yes, that was the plan earlier. because most of the unit tests might be dependent on the core implementation.
I agreed to add this ones since it looked like it would be more on HTML side.

Reply:
Had a conversation with Christian, he thought the same as this unit tests looks more on HTML side.
But still there are dependencies on a service to mock the method. hence, we decided It would be better to add unit tests while adding core implementation.